### PR TITLE
Adds support for organisation specific theme

### PR DESCRIPTION
**Pull request by dgparmar14**
# Description

This pull request addresses issue #249 by reorganizing the structure and configuration files related to personalized leaderboard colors.

## Changes Made

- Moved the `tailwind.config.js` file from the `leaderboard` repository to the `leaderboard-data` repository.
- Created a new `tailwind.config.js` file in the `leaderboard-data` repository.
- Established a symbolic link (symlink) between the `tailwind.config.js` file in the `leaderboard-data` repository and the `leaderboard` repository.
- Renamed the `load-fixtures.sh` script to `load-org-data.sh` for clarity and consistency.
- Updated the `load-org-data.sh` script to include the logic for fetching the `tailwind.config.js` file from the `leaderboard-data` repository.

## How It Works

After cloning the repository, users can run the command `./load-org-data.sh`, which fetches the data and `tailwind.config.js` file from the `leaderboard-data` repository, enabling personalized leaderboard colors.

This restructuring improves organization and maintainability while ensuring consistent behavior across environments.

Fixes: #249

### CODE REVIEW

**Comment by dgparmar14** [comment]
@rithviknishad Hey, rithvik it is right solution or any changes needed?

**Comment by dgparmar14** [comment]
@rithviknishad Hey, Rithvik now check it is it ok?

Screenshot 2024-02-29 224137
Screenshot 2024-02-29 224158

File: components/gh_events/GitHubEvents.tsx [hunk 1]
```diff
@@ -27,17 +26,29 @@ export default function GitHubEvents({ minimal }: { minimal?: boolean }) {
...
+          0,
+          5,
+        );
+        setEvents(combinedEvents);
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      }
+    };
+
+    fetchEvents();
```
Comment (rithviknishad) [inline components/gh_events/GitHubEvents.tsx:51]: Why were these modified?

File: tailwind.config.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
-/** @type {import('tailwindcss').Config} */
+/** @type {import("tailwindcss").Config} */
+import colorPallete from "./data-repo/colorPalette.json";
+
```
Comment (rithviknishad) [inline tailwind.config.js:3]: The JS Doc type comment belong to config and not color pallete

```suggestion
import colorPallete from "./data-repo/colorPalette.json";

/** @type {import("tailwindcss").Config} */
```

File: load-org-data.sh [hunk 3]
```diff
@@ -10,5 +10,7 @@ git config core.sparseCheckout true
 
 echo "data/" >> .git/info/sparse-checkout
 echo "contributors/" >> .git/info/sparse-checkout
+echo "colorPalette.json" >> .git/info/sparse-checkout
+
+git pull origin main
```
Comment (rithviknishad) [inline load-org-data.sh:15]: Why was this added?

```suggestion
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: components/gh_events/GitHubEvents.tsx (hunk 1 above)
Comment (dgparmar14) [reply components/gh_events/GitHubEvents.tsx:51]: Ohh!, I forgot to mention that .I noticed that there is use old method to fetch api .
So I use new and I thought for this small changes there is no need to raise issue so i changed it . Should I revert it or it's okk?

File: load-org-data.sh (hunk 3 above)
Comment (dgparmar14) [reply load-org-data.sh:15]: git pull origin main is already in .sh file

Comment (rithviknishad) [reply load-org-data.sh:15]: oh sorry, my bad

File: components/gh_events/GitHubEvents.tsx (hunk 1 above)
Comment (rithviknishad) [reply components/gh_events/GitHubEvents.tsx:51]: Revert and keep it as a separate PR 😄 

- Would be easier to review and test. 
- If something is wrong, it'd be easier to pin-point from where related issue/bug came from.
- Also gives you more points in PRs opened/merged in the leaderboard! 😜

Comment (rithviknishad) [reply components/gh_events/GitHubEvents.tsx:51]: Raise issue for it too, so that one could know what problem you are trying to solve by looking at the issue

File: tailwind.config.js (hunk 2 above)
Comment (dgparmar14) [reply tailwind.config.js:3]: Okay, I need to swap it right ?🤔
and what is the name of file ? colors or themes.json?

File: components/gh_events/GitHubEvents.tsx (hunk 1 above)
Comment (dgparmar14) [reply components/gh_events/GitHubEvents.tsx:51]: Okay I will raise a separate issue and separate pr for this. 😅Btw,  Thanks nice idea 💡

File: tailwind.config.js (hunk 2 above)
Comment (rithviknishad) [reply tailwind.config.js:3]: No changes in file name. The JS Doc type comment belongs to the exported config variable. just place the the type comment for the config. 

https://www.jetbrains.com/help/webstorm/creating-jsdoc-comments.html#ws_js_jsdoc_typescript_syntax

**Comment by dgparmar14** [comment]
@rithviknishad Hey,rithvik can you verify it?

**Review by rithviknishad — APPROVED** [review]
Awesome!
